### ISSUE TITLE
Confusing documentation on random functions

### ISSUE THREAD

**Issue by knuesel** [maths/moodle-qtype_stack#576]
On https://github.com/maths/moodle-qtype_stack/blob/master/doc/en/CAS/Random.md it says

> it is important not to use Maxima's own random() function. STACK creates pseudo-random numbers from a definite seed. This ensures that when a particular student returns they see the same variant of the question. Hence, STACK provides its own function rand().

But a bit later it says

> There are also Maxima's random functions. For example, to create a random list use random_permutation.

This seems like a contradiction. For example isn't `random_permutation` implemented using `random`? So if `random` must not be used, the same should apply to `random_premutation`.

**Comment by sangwinc** [comment]
Thanks for raising this and sorry about the confusion.

In the "mists of time" (not quite last century, but 10 years or so ago) I wrote a linear recurrence relation to generate pseudo-random numbers.  This was to ensure STACK took responsibility for maintaining random numbers from version to version.  But, people wanted to use other random functions provided by Maxima within STACK, so we switched back to relying on Maxima's random() command, and setting the seed yourself.

The downside is that if Maxima changes its random function, we will not be able to reproduce students' random versions.  So, beware upgrading your Maxima mid-way through a semester.

Anyway, sorry about the confusion and thank you for prompting me to review the documentation.
